examples/Swift SDK/Swift SDK.xcodeproj/project.pbxproj

```diff
@@ -193,6 +193,13 @@
 		B206F2432B7ACAFA0006C554 /* BaseDeeplinkTests.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = BaseDeeplinkTests.swift; sourceTree = "<group>"; };
 		B206F2882B7ACBBC0006C554 /* testInfo.plist */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.xml; path = testInfo.plist; sourceTree = "<group>"; };
 		B206F2892B7ACBBC0006C554 /* testInfoMissingCallbacks.plist */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.xml; path = testInfoMissingCallbacks.plist; sourceTree = "<group>"; };
+		B29302FD2B890D5800D28BAA /* UberSampleApp.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = UberSampleApp.swift; sourceTree = "<group>"; };
```

Comment:
Is there any plans to move this project to bazel?

Reply:
We do not have plans. And I don't think it would make things easy for 3P adopters.